### ISSUE TITLE
Remove Document, Package, File and Snippet keys in JSON and YAML

### ISSUE THREAD

**Issue by tsteenbe** [spdx/spdx-spec#157]
In the JSON and YAML format the current proposal for SPDX 2.2 is to encapsulate every SPDX Document, Package, File and Snippet with it own key. My proposal is to remove the unneeded keys from the output as they are not needed in JSON and YAML and make creation and parse more complex.

See this [example file](https://github.com/spdx/spdx-spec/blob/b1e7a206186351b88297a4776b45ccde88111f53/examples/SPDXJSONExample-v2.0.json)

```
{
  "Document" : {
    "spdxVersion" : "SPDX-2.0",
    "creationInfo" : {
```

```
          "File" : {
            "name" : "./src/org/spdx/parser/DOAPProject.java",
            "id" : "SPDXRef-DoapSource",
            "fileTypes" : [ "fileType_source" ],
            "checksums" : [ {
              "algorithm" : "checksumAlgorithm_sha1",
              "value" : "2fd4e1c67a2d28fced849ee1bb76e7391b93eb12"
            } ],
```

**Comment by tsteenbe** [comment]
@goneall  One more inconsistency for you to fix as noted in the SPDX meeting on November 19th, 2019 the current [example file] has two ways to represent SPDX FILE.

[Lines 22-33](https://github.com/spdx/spdx-spec/blob/b1e7a206186351b88297a4776b45ccde88111f53/examples/SPDXJSONExample-v2.0.json#L22-L33)
```
    "documentDescribes" : [ {
      "files" : [{
        "name" : "./package/foo.c",
        "id" : "SPDXRef-File",
        "fileTypes" : [ "fileType_source" ],
        "checksums" : [ {
          "algorithm" : "checksumAlgorithm_sha1",
          "value" : "d6a770ba38583ed4bb4525bd96e50461655d2758"
        }, {
          "algorithm" : "checksumAlgorithm_md5",
          "value" : "624c1abb3664f4b35547e7c73864ad24"
        } ],
```

[Lines 111 - 119](https://github.com/spdx/spdx-spec/blob/b1e7a206186351b88297a4776b45ccde88111f53/examples/SPDXJSONExample-v2.0.json#L111-L119)
```
        "files" : [ {
          "File" : {
            "name" : "./src/org/spdx/parser/DOAPProject.java",
            "id" : "SPDXRef-DoapSource",
            "fileTypes" : [ "fileType_source" ],
            "checksums" : [ {
              "algorithm" : "checksumAlgorithm_sha1",
              "value" : "2fd4e1c67a2d28fced849ee1bb76e7391b93eb12"
            } ],
```

**Comment by kestewart** [comment]
Gary agrees a bug, and should be fixed.

**Comment by goneall** [comment]
Resolved with PR #197
